### PR TITLE
ignore built files

### PR DESCRIPTION
**Pull request by SummerWish**
#206

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,15 @@
 *~
 test/node_modules/*
 test/coverage
+Help/new.html
+Help/content_server.html
+History/new.html
+Index/new.html
+Login/new.html
```
Comment (smcgregor) [inline .gitignore]: */*.html would be better than listing all these explicitly?

**Comment by smcgregor** [comment]
1. This pull request establishes the _requirement_ that people run the build system before they can contribute to the project.
2. People will not easily check the changes they make to built HTML by looking at the diff. This could hide errors.

Still, this is a change we need to make. I just need to think about the two points I raise above.

File: .gitignore (hunk 1 above)
Comment (SummerWish) [inline .gitignore]: There may be pages not built from the template but ends with `.html` in the future?

Comment (smcgregor) [inline .gitignore]: Tomorrow's problem.

**Comment by SummerWish** [comment]
Hello, what's your idea of merge this PR?

Currently I have to repeatly reset and remove my built HTML files if I want to switch my privly-application branch into another, which is not very convenient..Lazy is virtue for programmers :-)

**Comment by smcgregor** [comment]
This pull request requires me to write documentation in ~4 different repositories. You might say I am being very virtuous. If this is a high priority for you I can move it higher in the PR queue, but I am currently allocating my time where I think it is most needed

**Comment by SummerWish** [comment]
You are right. You should put efforts on more important things :-) I can bear this little inconvenience

**Comment by smcgregor** [comment]
See #206

**Pull request closed without merging**